app/src/main/java/ch/sdp/vibester/auth/FireBaseAuthenticator.kt

```diff
@@ -84,6 +84,17 @@ class FireBaseAuthenticator @Inject constructor() {
         return Firebase.auth.currentUser
     }
 
+    /**
+     * Getter for the current user ID
+     */
+    fun getCurrUID(): String {
+        var uid = ""
+        if (isLoggedIn()) {
+            uid = FirebaseAuth.getInstance().currentUser!!.uid
+        }
+        return uid
+    }
```

Comment:
```suggestion
```
See  40 lines above !!! 
getCurrentUID() exactly the same 😂

Reply:
yes, but when you mock, you cant access one from the above

Reply:
I will mock others as well, and remove those from companion object. sounds good?

Reply:
Ohh okay I see, yes maybe removing the companion object may be a good idea because I find it weird otherwise to have two times the same code.

Reply:
It is... I decided to remove one by one, so it easier to check!

Reply:
I will remove these function on the next pr since there a lot of dependencies. My next pr (where these repetetive functions are removed) is almost done!